src/group_theory/subgroup/basic.lean

```diff
@@ -2788,7 +2790,7 @@ end monoid_hom
 
 namespace mul_equiv
 
-variables {H K : subgroup G}
+variables {G' E : Type*} [group G'] [mul_equiv_class E G G'] {H K : subgroup G}
```

Comment:
I think this is a bad idea, because `subgroup.map` isn't generalized either. More concerning, I think this makes it much harder to choose a reasonable simp-normal form. Until we find a way to generalize `symm` over this typeclass, I think it does more harm than good here.